waterdata/tests/test_views.py

```diff
@@ -129,12 +83,101 @@ def test_5xx_from_water_services(self, r_mock):
     def test_agency_cd(self, r_mock):
         r_mock.return_value.status_code = 500
         response = self.app_client.get('/monitoring-location/{0}?agency_cd=USGS'.format(self.test_site_number))
-        r_mock.assert_called_with(self.test_hostname,
+        r_mock.assert_called_with(app.config['SERVICE_ROOT'],
                                   path='/nwis/site/',
                                   params={'site': self.test_site_number,
                                           'agencyCd': 'USGS',
                                           'siteOutput': 'expanded',
                                           'format': 'rdb'
-                                          }
-                                  )
+                                         }
```

Comment:
As was mentioned in https://github.com/usgs/waterdataui/pull/40, this was PEP8 compliant before...